Dockerfile.production

```diff
@@ -35,6 +35,7 @@ ENV INSTALL_PATH $INSTALL_PATH
 WORKDIR $INSTALL_PATH
 RUN gem install bundler
 
+COPY /db/ /usr/src/app/db/
```

Comment:
We shouldn't need to copy this directory over to the image path, did you run into problems if you did not copy it ?